Dockerfile

```diff
@@ -8,6 +8,11 @@ RUN apt-get update && apt-get install -y gfortran \
 
 RUN pip3 install git+https://github.com/GeoscienceAustralia/wofs --no-deps --global-option=build --global-option='--executable=/usr/bin/env python3'
 
+RUN pip3 install git+https://github.com/GeoscienceAustralia/eo-datasets@duuid_no_function --no-deps --global-option=build --global-option='--executable=/usr/bin/env python3'
```

Comment:
Do you think we should resolve the other PR first rather than reading from a branch?

Reply:
If we treat them independently it gives us more flexibility.  So I'm happy to keep them separate.